content/03.methods.md

```diff
@@ -5,8 +5,72 @@ Talk about dataset - Pubtator
 Talk about preprocessing Pubtator
 Talk about hand annotations for each realtion
 
-## Label Functions
-describe what a label function is and how many we created for each relation
+## Label Functions for Annotating Sentences
+A common challenge in natural language processing is having too few ground truth annotations, even when data are abundant.
+Data programming [@arxiv:1605.07723] circumvents this issue by quickly annotating large datasets by using multiple noisy signals emitted by label functions.
+Label functions are simple pythonic functions emit: a positive label (1), a negative label (-1) or abstain from emitting a label (0).
```

Comment:
Grammer:

> Label functions are simple pythonic functions emit

Do you mean "Pythonic" as in adhering to the ideal python idioms? Or do you just mean are written in Python?

Reply:
Could be both, but for our case these are functions written in python. Technically, label functions could be written in any desired language. Python is one of the easiest to deal with.